package.json

```
@@ -3,29 +3,33 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^13.0.0",
-    "@testing-library/user-event": "^13.2.1",
+    "@types/body-parser": "^1.19.2",
     "@types/node": "^18.7.11",
-    "@types/react": "^18.0.17",
+    "@types/react": "^18.0.19",
     "@types/react-dom": "^18.0.6",
     "axios": "^0.27.2",
-    "node-sass": "^4.13.1",
+    "chart.js": "^3.9.1",
+    "classnames": "^2.3.1",
+    "mobx": "^6.6.1",
+    "mobx-react-lite": "^3.4.0",
+    "paginate-array": "^2.1.0",
     "react": "^18.2.0",
+    "react-chartjs-2": "^4.3.1",
     "react-dom": "^18.2.0",
-    "react-router-dom": "6",
-    "react-scripts": "5.0.1",
+    "react-paginate": "^8.1.3",
+    "react-router-dom": "^6.3.0",
+    "sass": "^1.54.9",
     "styled-components": "^5.3.5",
-    "typescript": "^4.7.4",
+    "typescript": "^4.8.3",
     "web-vitals": "^2.1.0"
   },
   "scripts": {
-    "start": "react-app-rewired start",
-    "build": "react-app-rewired build",
-    "test": "react-app-rewired test",
-    "eject": "react-app-rewired eject",
+    "dev": "npx webpack serve && SET NODE_ENV=development",
+    "build": "npx webpack serve && SET NODE_ENV=production",
```

```suggestion
    "dev": "npx webpack serve --mode=development && SET NODE_ENV=development",
    "build": "npx webpack serve --mode=production && SET NODE_ENV=production",
```
это чтобы вебпак внутри себя делал разные оптимизации (вроде они такие есть). то есть SET NODE_ENV чисто для нас проставляет переменную процесса, чтобы мы могли ориентироваться, а для --mode даёт понять вебпаку, в каком окружении он находится